kahuna/public/js/edits/service.js

```diff
@@ -143,12 +145,12 @@ service.factory('editsService',
      * @param originalImage {Resource} the image used to check if we've re-indexed yet
      * @returns {Promise.<Resource>} completed when information is synced
```

Comment:
We should update the function description here to match the new param (as above re: description)